lib/messages.rb

```diff
@@ -4,7 +4,10 @@ class Message
       welcome: "Welcome to Tic Tac Toe!\n\n",
       instructions: "Tic Tac Toe is a two player game played\non a 3 x 3 grid. When prompted, each\nplayer will place their mark by entering\na number between 1-9 that corresponds with\nan available square on the grid.",
       get_spot: "Please select an available spot between 1-9:\n",
-      invalid_input: "That's not a valid spot.\n"
+      invalid_input: "That's not a valid spot.\n",
+      X_win: "Congratulations player X, you have won!\n",
+      O_win: "Congratulations player O, you have won!\n",
+      draw: "It's a draw!"
```

Comment:
I got a `%` at the end of the line when I got a draw (but not when a player wins):
<img width="343" alt="Screenshot 2023-04-17 at 7 00 48 PM" src="https://user-images.githubusercontent.com/73620216/232642304-62449fec-c37f-4493-bad6-c72273c47e99.png">

I see you have `\n` at the end of the other messages - maybe that would do the trick here too?

Reply:
Good catch! The next PR you see will make my messages more descriptive and tailored to specific players. But in my interest to get it working, I forgot to add the newline. I'll make the update now!